include/pmacc/PMaccConfig.cmake

```diff
@@ -141,6 +141,9 @@ if((NOT ALPAKA_ACC_GPU_CUDA_ENABLE) AND ALPAKA_ACC_GPU_CUDA_ONLY_MODE)
     message(WARNING "ALPAKA_ACC_GPU_CUDA_ONLY_MODE is set to OFF because cuda backend is not activated")
 endif()
 
+set(PMacc_LIBRARIES ${PMacc_LIBRARIES} ${cupla_LIBRARIES})
```

Comment:
double setting the includes does not harm, but yes with the target they can (should) be removed since the target already covers them which is also reducing a bit of code